.github/workflows/build-notebooks-TEMPLATE.yaml

```diff
@@ -46,6 +46,8 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      # region Free up disk space
```

Comment:
This is the VS Code-compatible syntax for collapsable regions. Allows to collapse parts of the file in IntelliJ and VS Code

![image](https://github.com/user-attachments/assets/f4d937da-0bf9-4a6d-ac0b-37e9692deda2)